Adds STARK as a type of ZK proof on layer-2-rollups doc (#5117)
ethereum/ethereum-org-website: src/content/developers/docs/scaling/layer-2-rollups/index.md

### Zero-knowledge rollups {#zk-rollups}

> **Zero-knowledge rollups (ZK-rollups)** bundle (or "roll-up") hundreds of transfers off-chain and generate a cryptographic proof. These proofs can come in a form of a SNARK (Succinct Non-interactive ARgument of Knowledge) or a STARK (Scalable Transparent ARgument of Knowledge). This is known as a validity proof and is posted on layer 1.

Reviewer: - I don't think we want to be so blunt about where the acronyms come from. It seems messy and doesn't add much in terms of understanding.
- Small grammar fix

```suggestion
**Zero-knowledge rollups (ZK-rollups)** bundle (or "roll-up") hundreds of transfers off-chain and generate a cryptographic proof. These proofs can come in the form of SNARKs (succinct non-interactive argument of knowledge) or STARKs (scalable transparent argument of knowledge). SNARKs and STARKs are known as validity proofs and get posted to layer 1.
```
